packages/round-manager/src/features/round/__tests__/RoundDetailForm.test.tsx

```diff
@@ -396,101 +396,3 @@ describe("<RoundDetailForm />", () => {
     expect(screen.getByTestId("chain-logo")).toBeInTheDocument();
   });
 });
```

Comment:
instead of creating a new file with the same content, can we `git mv ...` so we keep the history of the file?

Reply:
oh actually there are other tests in the original file so it makes sense like this, sorry I missed it!